rubix-spi/src/test/java/com/qubole/rubix/spi/TestCacheUtil.java

```diff
@@ -69,6 +74,24 @@ public void setUpConfiguration()
     conf.clear();
   }
 
+  private static String getHashedPath(String relLocation)
```

Comment:
This method should not be duplicated in the test class. You can mark the original method as `protected` and use the `@VisibleForTesting` annotation to signal the reason for the relaxed visibility.

Reply:
Done